acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/factory/PantheonNodeFactory.java

```diff
@@ -187,25 +189,16 @@ public PantheonNode createCliqueNode(final String name) throws IOException {
             .build());
   }
 
-  private Optional<String> createCliqueGenesisConfig(final Collection<RunnableNode> validators) {
-    String genesisTemplate = cliqueGenesisTemplateConfig();
-    String cliqueExtraData = encodeCliqueExtraData(validators);
-    String genesis = genesisTemplate.replaceAll("%cliqueExtraData%", cliqueExtraData);
-    return Optional.of(genesis);
-  }
-
-  private String encodeCliqueExtraData(final Collection<RunnableNode> nodes) {
-    final List<Address> addresses = nodes.stream().map(RunnableNode::getAddress).collect(toList());
-    return CliqueExtraData.createGenesisExtraDataString(addresses);
-  }
-
-  private String cliqueGenesisTemplateConfig() {
-    try {
-      URI uri = Resources.getResource("clique/clique.json").toURI();
-      return Resources.toString(uri.toURL(), Charset.defaultCharset());
-    } catch (URISyntaxException | IOException e) {
-      throw new IllegalStateException("Unable to get test clique genesis config");
-    }
+  public PantheonNode createIbftNode(final String name) throws IOException {
+    return create(
+        new PantheonFactoryConfigurationBuilder()
+            .setName(name)
+            .miningEnabled()
+            .setJsonRpcConfiguration(jsonRpcConfigWithClique())
```

Comment:
jsonRpcConfigWithClique? (not withIbft?)

Reply:
oh that's not right. i'll fix that

Reply:
done. it happen to be working because it is enabled by default. I still prefer to be explicit in this case though so am doing it the same as in the clique case and adding the rpc option.